test/functional/unified-spec-runner/entities.ts

```diff
@@ -222,7 +317,7 @@ export class EntitiesMap<E = Entity> extends Map<string, E> {
     for (const entity of entities ?? []) {
       if ('client' in entity) {
         const useMultipleMongoses =
-          config.topologyType === 'Sharded' && entity.client.useMultipleMongoses;
+          config.topologyType === 'LoadBalanced' && entity.client.useMultipleMongoses;
```

Comment:
why is this no longer desired for the sharded topologyType?

Reply:
Was a mistake - both sharded and load balanced are now checked here.